package.json

```diff
@@ -1,11 +1,19 @@
 {
     "name": "main",
-    "dependencies": {},
+    "dependencies": {
+        "patch-package": "^8.0.0"
+    },
     "workspaces": [
         "packages/reactivated",
         "packages/create-django-app",
         "sample",
         "website",
         "development"
-    ]
+    ],
+    "scripts": {
+        "postinstall": "patch-package"
+    },
+    "overrides": {
```

Comment:
Since we go rid of helmet, do we still need this?

Does it cascade down to consumer projects? If so, we *do* want that.